ftest/web-ui/webdriver/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "test": "nuxeo-web-ui-ftest --report --screenshots --headless",
     "test:dev": "nuxeo-web-ui-ftest --nuxeoUrl=http://localhost:8080/nuxeo --url=http://localhost:5000/",
-    "test:watch": "nuxeo-web-ui-ftest --nuxeoUrl=http://localhost:8080/nuxeo  --url=http://localhost:5000/ --tags=@watch --debug"
+    "test:watch": "nuxeo-web-ui-ftest --nuxeoUrl=http://localhost:8080/nuxeo --tags=@watch --debug"
```

Comment:
Despite the comments done on https://github.com/nuxeo/nuxeo-web-ui/pull/806, we should not change this parameter

Reply:
wasn't supposed to be erased, it's back to normal 👍